packages/flutter_tools/test/general.shard/generate_localizations_test.dart

```diff
@@ -743,7 +743,8 @@ void main() {
     );
   });
 
-  group('generateLocalizations', () {
+  group('generateLocalizations', ()
```

Comment:
Please remove all format-only diffs from this file, per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting

Reply:
Ah, raw texts confused me. Removed.